configure.ac

```diff
@@ -102,9 +102,6 @@ AC_ARG_ENABLE(CFD,
 AC_ARG_ENABLE(DOT,
    	AS_HELP_STRING([--disable-DOT], [build the SU2_DOT executable (default = yes)]),
 	[build_DOT=$enableval], [build_DOT="yes"])
-AC_ARG_ENABLE(MSH,
-   	AS_HELP_STRING([--disable-MSH], [build the SU2_MSH executable (default = yes)]),
-	[build_MSH=$enableval], [build_MSH="yes"])
```

Comment:
I wasn't a 100% sure I needed to do to this configure file. I just searched for MSH and deleted any offending lines. Let me know if something else needs to be done